include/alpaka/idx/MapIdx.hpp

```diff
@@ -64,8 +61,15 @@ namespace alpaka
 
                 // in-between
                 TElem hyperPlanesBefore = extent[lastIdx];
+#if BOOST_COMP_PGI && (defined(ALPAKA_ACC_ANY_BT_OMP5_ENABLED) || defined(ALPAKA_ACC_ANY_BT_OACC_ENABLED))
```

Comment:
```suggestion
#if BOOST_COMP_PGI && (defined(ALPAKA_ACC_ANY_BT_OMP5_ENABLED) || defined(ALPAKA_ACC_ANY_BT_OACC_ENABLED) && !defined(TPR30645))
```

This adds the internal bug number. Once the bug is fixed this macro will be defined and the workaround disabled.